smart-contracts/contracts/interfaces/IPublicLock.sol

```diff
@@ -288,19 +288,6 @@ interface IPublicLock {
     bytes calldata _data
   ) external payable;
 
-  /**
-   * This approves _approved to get ownership of _tokenId.
-   * Note: that since this is used for both purchase and transfer approvals
-   * the approved token may not exist.
-   * @param _approved The address to approve
-   * @param _tokenId The tokenId to approve
-   * @dev Throws if lock is disabled. Throws if called by other than key owner of =r approved address.
-   */
-  function approve(
```

Comment:
Why remove `approve`?

Reply:
See my comment below, bumy thinking is that Approve is already defined in IERC721.sol, so we shouldn't need it in our lock interface (as PublicLock.Sol inherits both IERC721.sol and IPublicLock.Sol.)